setup.py

```diff
@@ -108,9 +108,10 @@ def dmf_data_files(root: str = DMF_DATA_ROOT) -> List[Tuple[str, List[str]]]:
         "prerelease": DEPENDENCIES_FOR_PRERELEASE_VERSION,
         "optional": [
             "tensorflow",  # idaes.surrogate.keras_surrogate
+            "gridx-prescient==2.1",  # idaes.tests.prescient
```

Comment:
I'm not sure it matters, but we're fairly careful not to make breaking changes to Prescient at this point.
```suggestion
            "gridx-prescient>=2.1",  # idaes.tests.prescient
```

Reply:
Good point, I think it makes sense.